config/rbac/role.yaml

```diff
@@ -134,8 +134,6 @@ rules:
   - azuremachinepoolmachines/status
   verbs:
   - get
-  - patch
```

Comment:
I don't think you meant to do this.

Reply:
wowza, the generators might have removed this. sorry about that. re-added in https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/1460/commits/65ca5042956a85e50911c6a7f22d2975480921f6